.github/workflows/release_nightly.yml

```diff
@@ -133,29 +144,25 @@ jobs:
         with:
           ref: ${{ inputs.nightly_tag_main }}
 
-      - name: Install poetry
-        run: pipx install poetry==${{ env.POETRY_VERSION }}
-      - name: Set up Python 3.12
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          enable-cache: true
+          cache-dependency-glob: "uv.lock"
+      - name: "Set up Python"
         uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
-          cache: "poetry"
-      - name: Set up Nodejs 20
-        uses: actions/setup-node@v4
+          python-version-file: "pyproject.toml"
+      - name: Restore uv cache
+        uses: actions/cache@v4
         with:
-          node-version: "20"
-      - name: Verify Nightly Name and Version
```

Comment:
Can you add this back? Because the name and versions change from `main` and `base`, it's beneficial to check in both jobs.